tests/test_column.py

```diff
@@ -38,3 +38,7 @@ def test_expr_column_str_in_query(db: gp.Database, table: gp.Table):
         keys = list(row.keys())
         assert len(keys) == 1
         assert "id" in keys[0]
+
+
+def test_expr_column_rename(db: gp.Database, table: gp.Table):
```

Comment:
We also need to verify that the `rename()` returns a new object instead of modifying the old one.
try below
```
t = table["id"]
t_renamed = t.rename("table_id")
assert t is not t_renamed
assert str(t_renamed) == "const_table.id AS table_id"
```